packages/snap/src/configuration/predefined.ts

```diff
@@ -13,9 +14,11 @@ export const filecoinMainnetConfiguration: SnapConfig = {
 
 // replaces testnet for now
 export const filecoinDevnetConfiguration: SnapConfig = {
-  derivationPath: "m/44'/461'/0/0/1",
+  derivationPath: "m/44'/1'/0/0/1",
```

Comment:
do you have docs where this is defined?

Reply:
@mpetrunic I actually figured out from their code for deriving a key from seed that this `1` inside derivation path defines it is **testnet** address (snip below). But I will try now to find this inside documentation.

```ts
let testnet = false;
if (getAccountFromPath(path) === "1") {
  testnet = true;
}

function getAccountFromPath(path) {
  return path.split("/")[2].slice(0, -1);
}
```

Reply:
Usually thats suppose to be coin type.
https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#path-levels

Reply:
You should change 4th field to 1 and leave coin type as is

Reply:
But then the generated address is not for testnet (e.g. it resolves to `f13aw2za55qgpll6rxsebud5cl4fm3blr5no2xlpq` which is not testnet address as it doesn't start with `t`). 

I think that solution is to use `function keyRecover(privateKey, testnet)`, so after creating keypair with old derivation path `"m/44'/461'/0/0/1"` I can recover it for _testnet_ to obtain _testnet_ address.

I will write this code now, so you can have a clear idea about this solution.